docs/releases/5.2.txt

```diff
@@ -254,6 +254,10 @@ Tests
   failures easier to read and enables :option:`test --pdb` to directly enter
   into the failing test method.
 
+* Fixtures (including data from migrations enabled with
+  :ref:`serialized_rollback=True <test-case-serialized-rollback>`) are now
+  available during the ``setUpClass()`` phase of a ``TransactionTestCase``.
```

Comment:
> I don't think we need to mention serialized_rollback as from what I understand, that behavior hasn't changed

See above -- that's on me for not making this easier to reason about from the isolated tests.

> We might want to add a .. versionchanged:: 5.2 note to [TransactionTestCase.fixtures docs](https://docs.djangoproject.com/en/5.1/topics/testing/tools/#django.test.TransactionTestCase.fixtures) and check whether we can/should clarify this anywhere

Agree. Adam pointed out on the forum that people might happen to be relying on this data _not existing_ in their setUpClass, so it might need to go in miscellaneous breaking changes, too?

> This is essentially us treating this as a new feature rather than a big fix, we can also just remove this release note

I still think it's worth advertising: people who have fallen back to `setUp` might like to know they can unlearn the gotcha.

Reply:
On second thought I'm not convinced of the need to double-document it as a breaking change. (Thoughts?)

Reply:
I agree, a single note here is enough. The approach makes sense :+1: 

Maybe
```suggestion
* Data loaded from :attr:`~django.test.TransactionTestCase.fixtures`, and
  loaded from migrations enabled with :ref:`serialized_rollback=True
  <test-case-serialized-rollback>`, are now available during
  ``TransactionTestCase.setUpClass()``.
```